### DISCUSSION TITLE
ipv6 configuration on the primary load balancer

### DISCUSSION THREAD

**Discussion by HighSec-org** [community/community#13893] [Projects and Issues]
Hello,
I noticed something strange since yesterday.
It seems the ipv6 configuration of the load balancers for raw.githubusercontent.com seems to be broken today, and it was working earlier this week. I only noticed because my dev environment is ipv6 only. Ping returns the correct ipv6 address, but port 80 and 443 produces a 404.
I wasn't sure where to open a ticket about that.

**Comment by hmakmur** [comment]
I want to confirm this issue since it has not been fixed. I see ping doesnt even work..
It seems ipv6 is misconfigured or turned off despite the DNS registration 
This prevents any machines with ipv6 network to access raw.githubusercontent.com.

> % host raw.githubusercontent.com
> raw.githubusercontent.com has address 185.199.109.133
> raw.githubusercontent.com has address 185.199.110.133
> raw.githubusercontent.com has address 185.199.111.133
> raw.githubusercontent.com has address 185.199.108.133
> raw.githubusercontent.com has IPv6 address 2606:50c0:8002::154
> raw.githubusercontent.com has IPv6 address 2606:50c0:8003::154
> raw.githubusercontent.com has IPv6 address 2606:50c0:8000::154
> raw.githubusercontent.com has IPv6 address 2606:50c0:8001::154

> % ping  2606:50c0:8002::154
> PING 2606:50c0:8002::154(2606:50c0:8002::154) 56 data bytes
>  

> ^C
> --- 2606:50c0:8002::154 ping statistics ---
> 8 packets transmitted, 0 received, **100% packet loss,** time 7150ms

-hanz
